CMakeLists.txt

```diff
@@ -109,8 +109,10 @@ include_directories("vendor/fleece/Fleece"
 if(WIN32)
   set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /ignore:4099")
   include_directories("MSVC")
-elseif(UNIX AND NOT APPLE)
+elseif(ANDROID)
   include_directories("LiteCore/Android")
```

Comment:
This directory is now missing arc4random.h, so Android compilation fails (LiteCore/Unix can be safely included on Android, but not vice versa)

Reply:
Oopsies, of course.

Reply:
Done.